### PR TITLE
fraud docs

### PR DESCRIPTION
**Pull request by nickpatrick**

### CODE REVIEW

File: docs/fraud.mdx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+- `verified`: Indicates whether the request was made with SSL pinning (i.e., no man-in-the-middle attack)
+
+While you can work with individual flags, Radar also exposes a number of configurable flags to make your code simpler:
+- `passed`: Indicates whether all fraud checks passed
+- `bypassed`: Indicates whether a user's device has been bypassed for testing
+- `blocked`: Indicates whether a user's device has been manually blocked
+
+## Allowed states and countries
+
+You can mark specific states and countries as allowed or denied to comply with regulations. For example, a gambling app may only be allowed to operate in specific states.
```
Comment (brettguenther) [inline docs/fraud.mdx:73]: link to regions docs for now?

Comment (nickpatrick) [reply docs/fraud.mdx:73]: Yes

File: docs/fraud.mdx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+- `mocked`: Indicates whether a user's location is being mocked, such as in a simulator or using a location spoofing app (e.g., [Fake GPS location](https://play.google.com/store/apps/details?id=com.lexa.fakegps&hl=en_US&gl=US))
+- `compromised`: Indicates whether the user's device or app has been compromised according to the [Play Integrity API](https://developer.android.com/google/play/integrity) on Android
+- `jumped`: Indicates whether the user moved too far too fast (e.g., jumping across the country in only a few seconds)
+- `sharing`: Indicates whether the user is screen sharing (e.g., using [TeamViewer](https://www.teamviewer.com/en-us/))
+- `proxy`: Indicates whether the user's IP address is a known proxy, VPN, Tor exit node, etc.
+- `verified`: Indicates whether the request was made with SSL pinning (i.e., no man-in-the-middle attack)
+
+While you can work with individual flags, Radar also exposes a number of configurable flags to make your code simpler:
+- `passed`: Indicates whether all fraud checks passed
+- `bypassed`: Indicates whether a user's device has been bypassed for testing
```
[the remark this answers is not in the document]
Comment (nickpatrick) [reply docs/fraud.mdx:68]: Let me add a new section for this

File: docs/fraud.mdx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+## Fraud flags
+
+If you call `Radar.trackVerified()`, Radar exposes the following flags in `user.fraud`:
+- `mocked`: Indicates whether a user's location is being mocked, such as in a simulator or using a location spoofing app (e.g., [Fake GPS location](https://play.google.com/store/apps/details?id=com.lexa.fakegps&hl=en_US&gl=US))
+- `compromised`: Indicates whether the user's device or app has been compromised according to the [Play Integrity API](https://developer.android.com/google/play/integrity) on Android
+- `jumped`: Indicates whether the user moved too far too fast (e.g., jumping across the country in only a few seconds)
+- `sharing`: Indicates whether the user is screen sharing (e.g., using [TeamViewer](https://www.teamviewer.com/en-us/))
+- `proxy`: Indicates whether the user's IP address is a known proxy, VPN, Tor exit node, etc.
+- `verified`: Indicates whether the request was made with SSL pinning (i.e., no man-in-the-middle attack)
```
[the remark this answers is not in the document]
Comment (nickpatrick) [reply docs/fraud.mdx:64]: Let me link to the SSL pinning section for clarity

**Pull request merged**